README.md

```diff
@@ -146,6 +146,38 @@ custom:
 ```
 This will remove all folders within the installed requirements that match
 the names in `slimPatterns`
+
+### Lamba Layer
+Another method for dealing with large dependencies is to put them into a
+[Lambda Layer](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html).
+Simply add the `layer` option to the configuration.
+```yaml
+custom:
+  pythonRequirements:
+    layer: true
+```
+The requirements will be zipped up and a layer will be created automatically.
+Now just add the reference to the functions that will use the layer.
+```yaml
+functions:
+  hello:
+    handler: handler.hello
+    layers:
+      - {Ref: PythonRequirementsLambdaLayer}
```

Comment:
Would it make sense to set this automatically for python lambdas?

Reply:
I would say no since you may not want or need the layer in all of your lambda functions.

But maybe that could be controlled with an option? Either way, I'd say the default for using a layer should be opt-in.

Reply:
good enough reason for me 👍